<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>1249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1311</width>
      <height>1181</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Daily</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1281</width>
        <height>1131</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Monthly</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>741</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Yearly</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Summary</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame1sum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4"/>
     </widget>
     <widget class="QFrame" name="horizontalFrame2sum">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>691</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5"/>
     </widget>
     <widget class="QFrame" name="horizontalFrame3sum">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>490</y>
        <width>771</width>
        <height>611</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6"/>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1398</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
